client/ayon_hiero/api/plugin.py

```diff
@@ -860,15 +859,18 @@ def _convert_to_tag_data(self):
             # adding tag metadata from ui
             for _key, _value in self.pre_create_data.items():
                 if _key in self.tag_keys:
+                    # backward compatibility for reviewTrack (2024.11.07)
+                    if "reviewTrack" in _key:
+                        self.tag_data["reviewableSource"] = _value
                     self.tag_data[_key] = _value
 
             # driving layer is set as positive match
             if hero_track or self.vertical_sync:
                 # mark review layer
-                if self.review_track and (
-                        self.review_track not in self.review_track_default):
+                if self.review_source and (
+                        self.review_source is not self.review_source_default):
```

Comment:
If this is string comparison it should not be `is not`
```suggestion
                        self.review_source != self.review_source_default):
```